### DISCUSSION TITLE
Antenna Diagnostics for damaged, missing, or incorrect antenna

### DISCUSSION THREAD

**Discussion by vtjballeng** [jgromes/RadioLib#680] [Ideas]
Is there any known way to determine if no antenna is connected on an SX1262? If this is possible, is it also possible to have performance baselines or sweep test diagnostics to determine is a damaged or inappropriate antenna is used?

**Comment by jgromes** [comment]
I'm not aware of any functionality within the SX1262 that would allow this simply, the only thing I can think of is checking a known local signal source - but then one would have to known the expected signal strength at the receiver to be able to decide whether the antenna is good or not.
